io.github.Samuel_Schlemper_Schlemuel.SingWriter.metainfo.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 Samuel Schlemper -->
+<component type="desktop-application">
+
+<id>io.github.Samuel_Schlemper_Schlemuel.SingWriter</id>
+<metadata_license>CC0-1.0</metadata_license>
+<project_license>GPL-3.0-or-later</project_license>
+
+<name translate="no">SingWriter</name>
+<summary>Write in SignWriting</summary>
+<summary xml:lang="pt_BR">Escreva em SignWriting</summary>
+
+<developer id="io.github.Samuel_Schlemper_Schlemuel">
+  <name translate="no">Samuel Schlemper</name>
+</developer>
+
+<description>
+  <p>Write in SignWriting (the write form of sign languages (Ex: ASL and Libras)) and convert the text to PDF</p>
+  <p xml:lang="pt_BR">Escreva em SignWriting (a forma escrita das línguas de sinais (Ex: ASL e Libras)) e converta o texto para PDF</p>
+  <p>The application uses the open font Noto Sans SignWriting. If your computer does not have this font, the application will not work properly. Install the font before using the application.</p>
+  <p xml:lang="pt_BR">O aplicativo usa a fonte aberta Noto Sans SignWriting. Se seu computador não tem essa fonte, o aplicativo não ira funcionar corretamente. Instale a fonte antes de usar o aplicativo.</p>
+</description>
+
+<launchable type="desktop-id">org.flatpak.gnome.desktop</launchable>
+
+<icon type="stock">io.github.Samuel_Schlemper_Schlemuel.SingWriter.svg</icon>
+<icon type="remote">https://raw.githubusercontent.com/Samuel-Schlemper-Schlemuel
+/SingWriter/f3bb637495e2d2ed384992dd5b1c187df62cfeed/data/icons/hicolor/scalable/apps
+/io.github.SamuelSchlemperSchlemuel.SingWriter.svg</icon>
+
```

Comment:
```suggestion
```

This will be the third time I request this change.

Reply:
Sorry, I thought you had asked for a line break, I didn't realize you wanted me to remove them.